src/Forms/EditFormFactory.php

```diff
@@ -103,4 +100,22 @@ protected function namespaceFields(FieldList $fields, array $context)
             $field->setName($namespacedName);
         }
     }
+
+    /**
+     * Remove the pseudo namespaces that were added in namespaceFields()
+     */
+    public function removeNamespaceFromFields(FieldList $fields, array $context): void
```

Comment:
Using the same method signature as the corresponding `namespaceFields()` since they're essentially the reverse of one another.